apps/emqx_bridge_oracle/test/emqx_bridge_oracle_SUITE.erl

```diff
@@ -196,7 +196,6 @@ oracle_config(TestCase, _ConnectionType, Config) ->
         io_lib:format(
             "bridges.oracle.~s {\n"
             "  enable = true\n"
-            "  database = \"~s\"\n"
```

Comment:
Nit: maybe add `service_name` to assert it's a valid config as expected.

Reply:
we use `service_name` for Oracle RAC, which I believe it's not supported by oracleinanutshell. Therefore, I'm setting only `sid`. Also, both `sid` and `service_name` are optionals, so missing `service_name` here is valid =)

Reply:
I see.  I was suggesting adding `service_name` here to assert that it is a valid field.

Reply:
After discussing, I understood that we can't test this field with the current oracle test container.